### DISCUSSION TITLE
error with ternary filter e summarize

### DISCUSSION THREAD

**Discussion by newtec-foggia** [filamentphp/filament#9605] [Help] [answered]
if I use the ternary filter and the count with summarize in the resource it reports an error , how to ?

thanks !

![Screenshot 2023-11-12 184825](https://github.com/filamentphp/filament/assets/83398776/046776a4-ac1f-425f-916e-2427234acc83)
![summarize function](https://github.com/filamentphp/filament/assets/83398776/1b3d9ce2-641f-4197-8d34-fe7f84693c12)
![ternary](https://github.com/filamentphp/filament/assets/83398776/fb55409f-53cf-4f3e-977b-8f96cb53c089)

**Comment by leandrocfe** [comment, marked as answer]
```php
use Illuminate\Database\Eloquent\Builder as EloquentBuilder;

TernaryFilter::make('saldo')
->queries(
true: fn(EloquentBuilder $query)...
)
```

**Reply by newtec-foggia** [reply]
It worked perfectly, thank you very much @leandrocfe
